packages/cyberstorm/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import React, { CSSProperties, ReactNode } from "react";
+import styles from "./Toast.module.css";
+import { Icon } from "../Icon/Icon";
+import { classnames } from "../../utils/utils";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faXmarkLarge } from "@fortawesome/pro-solid-svg-icons";
+import * as RadixToast from "@radix-ui/react-toast";
+
+type _ToastProps = {
+  id: string;
+  content: ReactNode;
+  icon?: JSX.Element;
+  variant?: "info" | "danger" | "warning" | "success";
+  timer?: number;
```

Comment:
This could be `duration` so it would match the terms used by Radix (also IMO it's a better name anyway, I wasn't sure what timer referred to without checking the code).

Also does override the `duration` prop in the provider? The one which is currently hardcoded to 10000. What happens if I pass a conflicting value here?

Reply:
Yep https://www.radix-ui.com/primitives/docs/components/toast#custom-duration